sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -143,8 +143,9 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     }
 
     try {
+      const urlSuffix = tenantId === "adfs" ? "oauth2/token" : "oauth2/v2.0/token";
```

Comment:
We should definitely pull this out into a helper method now so that we don't have to maintain it in 4 different places.

Reply:
moved it as a helper under util

Reply:
Thanks!  Looks great